### ISSUE TITLE
Cannot view a configuration by itself it it is a prefix

### ISSUE THREAD

**Issue by rakudrama** [dart-lang/dart_ci#144]
I want to view `dart2js-hostasserts-linux-d8` by itself but there seems to be no way to do that and I always get `dart2js-hostasserts-linux-d8-unsound` as well.

I tried putting the configuration in quotes and ending it with `$`, but these did not work.

**Comment by whesse** [comment]
No, there is no current way to do this.  The code that interprets a filter term as a configuration prefix, and gets all configs starting with it, is at 

https://github.com/dart-lang/dart_ci/blob/main/current_results/lib/src/slice.dart#L169

for terms like "configuration:foo" 
and below that for terms without "configuration:".

We could change the meaning of "configuration:foo" to be an exact match, while keeping prefix matching for filter terms without a colon.
